<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Screen_Cad</class>
 <widget class="QMainWindow" name="Screen_Cad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(66, 198, 198);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Button_Back">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>430</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 127);
font: 25pt &quot;Niagara Solid&quot;;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Voltar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Input_Name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: groove;
border-width: 3px;
border-color: rgb(0, 0, 0);
border-radius: 15px;
background-color: rgb(241, 241, 241);
font: 20pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Ornament_Name">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>10</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Nome de usu√°rio</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Ornament_Pass_Confirm">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Confirme a senha</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Ornament_Pass">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Senha</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Ornament_Email">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Input_Pass_Confirm">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: groove;
border-width: 3px;
border-color: rgb(0, 0, 0);
border-radius: 15px;
background-color: rgb(241, 241, 241);
font: 20pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Input_Pass">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: groove;
border-width: 3px;
border-color: rgb(0, 0, 0);
border-radius: 15px;
background-color: rgb(241, 241, 241);
font: 20pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Input_Email">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: groove;
border-width: 3px;
border-color: rgb(0, 0, 0);
border-radius: 15px;
background-color: rgb(241, 241, 241);
font: 20pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Warning_Same_Pass">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Times New Roman&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Cad">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 127);
font: 25pt &quot;Niagara Solid&quot;;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Cadastrar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
